tomviz/DataTransformMenu.cxx

```diff
@@ -132,12 +133,14 @@ void DataTransformMenu::buildTransforms()
   new AddPythonTransformReaction(unsharpMaskAction, "Unsharp Mask",
                                  readInPythonScript("UnsharpMask"), false,
                                  false, readInJSONDescription("UnsharpMask"));
-  new AddPythonTransformReaction(laplaceFilterAction, "Laplace Filter",
+  new AddPythonTransformReaction(laplaceFilterAction, "Laplace Sharpen",
                                  readInPythonScript("LaplaceFilter"));
-  new AddPythonTransformReaction(gaussianFilterAction, "Gaussian Filter",
-                                 readInPythonScript("GaussianFilter"), false,
-                                 false,
-                                 readInJSONDescription("GaussianFilter"));
+  new AddPythonTransformReaction(autoCorrelationAction,"Auto Correlation",
+                                 readInPythonScript("Auto_Correlation"));
+  new AddPythonTransformReaction(gaussianFilterAction, "Gaussian Blur",
+                                   readInPythonScript("GaussianFilter"), false,
```

Comment:
Indentation looks off in these lines, should be inline with the g of gaussianFilterAction